ui/web-v2/apps/admin/src/grpc/utils.ts

```diff
@@ -6,6 +7,7 @@ import { getToken } from '../storage/token';
 
 type MetaData = {
   authorization: string;
+  'accept-language': LanguageTypes;
```

Comment:
To keep consistency, please add the quotes to the `authorization` field also. Or remove them from `accept-language`.

Reply:
@cre8ivejp  Quotes is added because 'accept-language' has minus (-) symbol and we can not add quote in authorization. It's a javascript object key.